widgets/toggle/toggle.js

```diff
@@ -68,11 +59,7 @@ function toggle({
         transformData: prepareToggleData(transformData),
         defaultTemplates,
         templatesConfig,
-        templates: {
-          header: templates.header,
-          item: templates.body,
```

Comment:
Yes indeed, I tried to be smart. This will be a BREAKING CHANGE (only for history/search, because nobody uses this :))